### PR TITLE
Add event Journey Into the World of Service Meshes and Meshery 

### LINKED ISSUE #2422: [Event] Journey Into the World of Service Meshes and Meshery
Please add the following event to layer5.io/events.

Event:

    Event Title: Journey Into the World of Service Meshes and Meshery
    Event Link: sched.co/siEs
    Talk Description: See the event link
    Speakers: Nithish Karthik, Aditya Krishna Sharma

Contributor Resources

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the master branch.

    See contributing instructions
    See Layer5 site designs in this Figma project.

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
Signed-off-by: Prakhar Agarwal <prakhar@everything.net.in>

**Description**

This PR fixes #2422

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e04f861302557e94944fba6f8ff92405c2023d25 at: https://61bd62287cbfaf9cc2dcbf78--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 23b5eeeca2e94bf2c78fbd281e6a0eb6bdce5acf at: https://61bd670c9c4a8db299be1f0b--layer5ng.netlify.app

File: src/collections/events/service-mesh-and-meshery/index.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Journey Into the World of Service Meshes and Meshery"
+date: 2022-01-29
+thumbnail: ./service-mesh-and-meshery.png
+eurl: https://sched.co/siEs
+published: true
+upcoming: true
+speakers: ["Nithish Karthik", "Aditya Krishna Sharma"]
```
Comment (leecalcote) [inline src/collections/events/service-mesh-and-meshery/index.mdx:8]: Are we missing an "h", @adithyaakrishna?

Comment (leecalcote) [inline src/collections/events/service-mesh-and-meshery/index.mdx:8]: @Prakhar-Agarwal-byte these name should match existing contributors names such that the name becomes a hyperlink to the community member profile. Do you know if that was the case when you built it?

Comment (Prakhar-Agarwal-byte) [reply src/collections/events/service-mesh-and-meshery/index.mdx:8]: I copied the names from the issue itself. The name mentioned in [contributor profile](https://61bd62287cbfaf9cc2dcbf78--layer5ng.netlify.app/community/members/adithya-krishna) is "Adithya Krishna"

**Comment by leecalcote** [comment]
@Prakhar-Agarwal-byte will you make this a generic devconf event listing so that the other devconf talk can be consolidated into it?

File: src/collections/events/service-mesh-and-meshery/index.mdx (hunk 1 above)
Comment (adithyaakrishna) [reply src/collections/events/service-mesh-and-meshery/index.mdx:8]: Yep, we are actually 😂 
Thanks for pointing it out @leecalcote ❤️

Comment (adithyaakrishna) [reply src/collections/events/service-mesh-and-meshery/index.mdx:8]: @Prakhar-Agarwal-byte You can change it to the one mentioned in the profile itself, no problem :)

**Comment by adithyaakrishna** [comment]
@Prakhar-Agarwal-byte You can refer to this PR, https://github.com/layer5io/layer5/pull/2425/files regarding @leecalcote 's comment above

**Comment by leecalcote** [comment]
@adithyaakrishna do you mind finishing off this PR? @Prakhar-Agarwal-byte hasn't responded here or in Slack.

**Comment by leecalcote** [comment]
@Abhijay007 please move forward with consolidation. This PR appears to be abandoned.

**Comment by leecalcote** [comment]
Reopening this PR to get an initial version of this out. Hopefully, @navendu-pottekkat responds in time to get the other PR completed and consolidated.

**Comment by l5io** [comment]
🚀 Preview for commit 23b5eeeca2e94bf2c78fbd281e6a0eb6bdce5acf at: https://61d37b89c2aedf005aa6cfff--layer5ng.netlify.app

**Pull request merged**